packages/client/src/main.css

```diff
@@ -1054,3 +1088,57 @@ decorator-lens.active {
 [aria-expanded='true'] .menuSearch__line-4 {
   transform: translate3d(-3px, -3px, 0);
 }
+
+/* Modal */
+
+.modal {
```

Comment:
Burde kanskje bruke CSS modules her for å unngå scoping problemer. Men det kan jeg fikse!